xml/tuning_cgroups.xml

```diff
@@ -591,7 +591,7 @@ io
 [...]
 </screen>
 <para>
-    Each process gets half of the throughput for I/O operations. Set up two control groups, one for each process, verify that BFQ is 
+    Each process gets half of the throughput for I/O operations. Set up two control groups&mdash;one for each process&mdash;verify that BFQ is 
```

Comment:
I wonder why the dash is replaced only here, or is it due to the editor you are using?